docker-compose.yml

```diff
@@ -25,6 +25,9 @@ services:
       - CORS_ORIGIN_WHITELIST=http://localhost:8080,https://preview.clock.uni-frankfurt.de
       - GOETHE_OAUTH2_REDIRECT_URI=https://preview.clock.uni-frankfurt.de/logging-in
       - SITE_ID=1
+      - IDM_API_URL=default
+      - IDM_API_KEY=default
+      - IDM_SECRET_KEY=64656661756c74
```

Comment:
Dürfen wir den hier so veröffentlichen oder kommt das besser in ein secret?

Reply:
Das ist natürlich nicht der echte Secret sondern nur ein Dummy filler Key :D